xbmc/utils/DiscsUtils.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ *  Copyright (C) 2022- Team Kodi
```

Comment:
intended `2022-` ?

Reply:
It was. Isn't it the recommended way for new files?

Reply:
no idea 😅

Reply:
The years should be a list of individual and ranges, and a range has two have two years on either side of the dash. When I create new files in 2022, I do

```c++
/*
 *  Copyright (C) 2022 Team Kodi
 ```